packages/core/src/components/button/_button-group.scss

```diff
@@ -147,7 +146,6 @@ Styleguide components.button-group.css
 
       &::after {
         @include pt-button-minimal-divider();
-
```

Comment:
i've always found this whitespace helpful in selectors with lots of properties. 
is it now banned by stylelint?